beacon-chain/forkchoice/protoarray/unrealized_justification.go

```diff
@@ -0,0 +1,65 @@
+package protoarray
+
+import (
+	types "github.com/prysmaticlabs/prysm/consensus-types/primitives"
+)
+
+func (s *Store) setUnrealizedJustifiedEpoch(root [32]byte, epoch types.Epoch) error {
+	s.nodesLock.Lock()
+	defer s.nodesLock.Unlock()
+	index, ok := s.nodesIndices[root]
+	if !ok {
+		return ErrUnknownNodeRoot
+	}
+	if index >= uint64(len(s.nodes)) {
+		return errInvalidNodeIndex
+	}
+	node := s.nodes[index]
```

Comment:
Check ok too?

```suggestion
	node, ok := s.nodes[index]
```

Reply:
It's impossible for that to happen given the previous check.